expr/sort.go

```diff
@@ -180,7 +180,7 @@ func lookupSorter(typ zng.Type) comparefn {
 			return -1
 		}
 
-	case zng.TypeString, zng.TypeEnum:
+	case zng.TypeString:
```

Comment:
Tangential to this pr but does this work for aliases? Are the values unwrapped before they get here?

Reply:
I didn't follow the initial implementation of aliases closely.  I thought the desired behavior was that an aliased type is sorted separately?  From a quick glance, that looks to me like what `sort_test.go` is testing?  (though I may have mis-read it)